tests/testthat/test-csv.R

```diff
@@ -888,3 +888,11 @@ test_that("read_cmdstan_csv works if no variables are specified", {
     read_cmdstan_csv(fit_bernoulli_thin_1$output_files(), variables = "", sampler_diagnostics = "")
   )
 })
+
+test_that("read_cmdstan_csv() works with tilde expansion", {
+  skip_if(os_is_windows())
+  full_path <- test_path("resources", "csv", "model1-1-warmup.csv")
+  expect_no_error(read_cmdstan_csv(full_path))
+  tildified_path <- file.path("~", fs::path_rel(full_path, "~"))
```

Comment:
If there's a way to avoid using `fs::path_rel()` and adding `fs` to Suggests I guess that would be preferable, but I'm guessing it's pretty annoying to replicate `path_rel()` using base R? If that's the case no worries, we can add it to Suggests.